### PR TITLE
Documentation for the tools [skip tests]

### PR DESCRIPTION
**Pull request by hackaugusto**
- This also moved the kill_if_no_output.py to the .circleci folder,
  since the tool is only useful for the CI.
- It also makes sure every tool is executable and has a hash-bang, which
  uses the `/bin/env` instead of the system binaries.

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+tools
+-----
+
+- `parallel_tests.sh`: Tool to execute the test suite in parallel, useful to
+  run multiple integration tests at the same time.
+- `startcluster.py`: This tool will generate a genesis file with a few
+  pre-funded accounts to be used as Raidne accounts, and then run a local
+  Geth private chain, useful for debugging.
+- `genesis_builder.py`: This tool will generate a genesis file.
```
Comment (palango) [inline tools/README.md]: YOu link to this script here, but it's deleted.

**Comment by LefterisJP** [comment]
Lol I just got the notificaiton to review this PR even though auto-assign assigned my yesterday. The fuck?

File: tools/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+tools
+-----
+
+- `parallel_tests.sh`: Tool to execute the test suite in parallel, useful to
+  run multiple integration tests at the same time.
+- `startcluster.py`: This tool will generate a genesis file with a few
+  pre-funded accounts to be used as Raidne accounts, and then run a local
```
Comment (LefterisJP) [inline tools/README.md]: typo: Raidne -> Raiden

File: tools/README.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  Geth private chain, useful for debugging.
+- `genesis_builder.py`: This tool will generate a genesis file.
+- `transfer_eth.py`: This is just a simple tool to send a eth transfer using
```
Comment (LefterisJP) [inline tools/README.md]: a eth -> an eth

File: tools/README.md (hunk 1 above)
Comment (LefterisJP) [reply tools/README.md]: +1, nice catch

**Pull request merged**